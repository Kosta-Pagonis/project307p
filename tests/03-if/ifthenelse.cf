if (true)
  print_str("hello ");

if (false)
  print_int(5);
else
<<<<<<< HEAD
  print_str("world\n");
=======
  print_str("world");
>>>>>>> d68aeae57ff6fd3d10adab4bd85b031b409a0028
