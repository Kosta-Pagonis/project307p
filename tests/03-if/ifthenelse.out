<<<<<<< HEAD
hello world

=======
hello world
>>>>>>> d68aeae57ff6fd3d10adab4bd85b031b409a0028
